file.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"net/url"
+	"path/filepath"
+	"strings"
+)
+
+type file struct {
+	Name string
+	URL  string
+}
+
+type files []file
+
+func (fs files) String() string {
+	fileNames := []string{}
+	for _, file := range fs {
+		fileNames = append(fileNames, fmt.Sprintf("$%s/%s", envKey, file.Name))
```

Comment:
It should not, because the method will return something like this:
```
$GENERIC_FILE_STORAGE/install_wrap.sh
$GENERIC_FILE_STORAGE/run_wrap.sh
$GENERIC_FILE_STORAGE/removel_wrap.sh
```